inductiva/_cli/storage/storage_size.py

```diff
@@ -0,0 +1,18 @@
+"""Get the user's remote storage size in GB via CLI."""
+
+from inductiva import storage
+
+
+def storage_used(args):
+    """List the user's remote storage contents."""
+    del args
+    storage.get_space_used()
```

Comment:
this needs to be refactored in the future. The `storage.get_space_used()` call should return data and not print to stdout. Please open a task for this.

Reply:
I open an issued, but one inspection of the method, it actually returns the value and also prints it.